documentation/CAMARA-Security-Interoperability.md

```diff
@@ -105,6 +105,64 @@ The following table defines the REQUIRED behaviour of the API Provider for the `
 | No                               | Supports DPoP                     | Bearer token         |
 | No                               | Requires DPoP                      | HTTP 400 `invalid_dpop_proof`<br>(see RFC [9449](https://www.rfc-editor.org/rfc/rfc9449.html#name-oauth-extensions-error-regi)) |
 
+
+### Additional Recommendations for DPoP Implementations 
+
+For API providers (e.g., operators) that support and require the Demonstration of Proof-of-Possession (DPoP) mechanism as defined in [RFC 9449], the following measures MAY be used to strengthen message integrity and replay protection. 
+
+**Request Binding to DPoP Proofs**
+
+As per RFC 9449 §11.7, the DPoP proof binds the HTTP method (htm) and target URI (htu) but omits headers and the request body.  To prevent message alteration or tampering, API providers SHOULD require clients to include additional CAMARA-defined claims in the DPoP proof: 
```

Comment:
```suggestion
As per RFC 9449 [section 11.7](https://www.rfc-editor.org/rfc/rfc9449.html#section-11.7), the DPoP proof binds the HTTP method (htm) and target URI (htu) but omits headers and the request body.  To prevent message alteration or tampering, API providers SHOULD require clients to include additional CAMARA-defined claims in the DPoP proof: 
```